pkgs/development/python-modules/checksumdir/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, setuptools
+, wheel
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "checksumdir";
+  version = "1.2.0";
+  format = "pyproject";
+
+  disable = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "to-mc";
+    repo = "checksumdir";
+    rev = version;
+    hash = "sha256-PO8sRGFQ1Dt/UYJuFH6Y3EaQVaS+4DJlOQtvF8ZmBWQ=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    setuptools
+    wheel
```

Comment:
Do we need these dependencies?

Reply:
Tried it out, we do need `setuptools`, but we do not need `wheel`, good catch! Fixed